docs/source/dev_guide/algorithms/dbscan/computation-batch.rst

```diff
@@ -53,6 +53,9 @@ The DBSCAN clustering algorithm has the following parameters:
      - If flag is set to false, all neighborhoods will be computed and stored prior to clustering.
        It will require up to :math:`O(|\text{sum of sizes of all observations' neighborhoods}|)` of additional memory, 
        which in worst case can be :math:`O(|\text{number of observations}|^2)`. However, in general, performance may be better.
+
+       .. note:: ``memorySavingMode`` could be ``false`` only on CPU.
```

Comment:
Unclear formulation. False is default value. Does it mean that algorithm will fail, if default value to be used? Anyway, 'could be' is bad for technical doc, some specific should be explained, like parameter to be ignored o GPU, or exception to be raised.

Reply:
@amgrigoriev could you clarify this?

Reply:
Algorithm will return error if memorySavingMode is true. It was a requirement of reviewers. Original behavior was to ignore the parameter. It is handled in python. My suggestion "is supported" instead of "could be".

Reply:
I propose this description:
* On GPU, the ``memorySavingMode`` flag can only be set to ``true``. You will get an error if the flag is set to ``false``.

Reply:
Agree